app/src/main/kotlin/no/nav/k9punsj/omsorgspengeraleneomsorg/MapOmsAOTilK9Format.kt

```diff
@@ -103,15 +102,9 @@ internal class MapOmsAOTilK9Format(
 
     internal companion object {
         private val logger = LoggerFactory.getLogger(MapOmsAOTilK9Format::class.java)
-        private val Oslo = ZoneId.of("Europe/Oslo")
+
         private val Validator = OmsorgspengerKroniskSyktBarn().validator
```

Comment:
Hm, er dette for att vi ikke validerer noenting her? :)

Reply:
Samme i midlertidigalene

Reply:
Den brukes fortsatt, men den er refaktorert ut til ZoneUtils, og brukes ved mapping til K9Format i enkelte datastrukturerer.